src/virt-install

```diff
@@ -252,6 +252,14 @@ try:
     # to ensure we fully control the code.
     cleanup_argv = ['/usr/lib/coreos-assembler/gf-anaconda-cleanup', args.dest]
     run_sync_verbose(cleanup_argv)
+
+    # This one is part of the "API" we provide to config repositories.  At least
+    # RHEL CoreOS will use it to change the OS to appear to have been provisioned
+    # from an oscontainer.
+    config_image_post = os.path.join(args.configdir, "image-postprocess")
```

Comment:
Hmm, WDYT about just making this a field in the `image.yaml` instead?

I like the overall idea of cosa expecting some specific filenames in places so all the config repos look the same. But at the same time, it makes things harder to connect and grok. I've been thinking lately it'd be cool to have a single `assembly.yaml` file which points to the manifest, the `installer/` dir, and subsumes `image.yaml`.